part-4/model-building-components.md

```diff
@@ -418,63 +418,104 @@ Models generated in **PK-Sim**® make extensive **use of tags**: For example, op
 
 Similarly, observers or events can be included or excluded from being created in different parts of the spatial structure. The molecule observer "Fraction excreted", for example, makes use of the tag "Urine", so this observer is only created in the urine container.
 
-## Molecule Start Values‌
+# Initial Conditions Building Blocks
 
-Molecule start values are needed to define the initial amounts of all molecules present in the molecules building block used in a simulation for all containers. These values are either imported when loading a simulation, or they can be created automatically and edited manually, if needed.
+The Initial Conditions (IC) Building Block (BB) defines the containers in which the molecules will be present and their initial amounts.
 
-{% hint style="info" %}
-Molecule and Parameter Start Values can also be created within the Simulation Creation Wizard (compare [Create a Simulation](setting-up-simulation.md#create-a-simulation)).
-{% endhint %}
+The following section describes the functionalities of the IC BB on a PBPK model exported from PK-Sim. Later on, a simple [example](#example---creating-ic-bb) is given to create a new IC BB and populate it with information.
 
-To automatically create molecule start values by MoBi®:
+## Initial Conditions - Functionality‌ Overview
 
-1. Right-click the entry **Molecule Start Values** in the building block explorer.
-2. Select **Create Molecule Start Values Building Block** from the context menu that appears.
-3. A window called "Create new start values" opens. Enter a unique name for the building block.
-4. In the comboboxes below, you can select between different molecules or spatial structure building blocks from which the start values are calculated.
-5. Click **OK** or press **Enter**.
-6. If the name you have entered is already in use, you may be asked for entering a new name.
-7. An edit window opens, containing all created parameters.
+An IC BB can contain entries for molecules and physical containers across different modules.
+
+In contrast to other BB types except for the parameter values BB, multiple IC BBs can be created within one module. This allows you to define different initial conditions for different simulation scenarios. For example, different initial concentrations of an endogenous molecule may represent different disease states. During simulation creation, you can select which IC BB to use.
```

Comment:
_...**multiple** IC BBs can be created..._ 
(just putting multiple in bold)

Reply:
done